### PR TITLE
Add footer to the page

### PR DESCRIPTION
**Pull request by tarebyte**
Feedback welcome

/cc @johndbritton @cameronmcefee 

screen shot 2015-09-04 at 22 53 02

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -9,7 +9,3 @@
 @import "components/*";
 
 @import "layout";
-
-@import "header";
-
```
Comment (tarebyte) [inline app/assets/stylesheets/application.scss]: This is just leftover CSS that I didn't delete the first time around

**Comment by tarebyte** [comment]
Not sure how to include #178 into the footer

**Comment by johndbritton** [comment]
Maybe put #178 in the header using the bug octicon and a tooltip "Report an issue"

**Comment by johndbritton** [comment]
:+1: from me, maybe see if @cameronmcefee agrees.

**Comment by tarebyte** [comment]
@johndbritton the tooltip does weird stuff to the header, what do you think about just having "Report an Issue" as part of the header when logged in?

**Comment by cameronmcefee** [comment]
:+1: looks good to me. Depending on your goals with icons, you can use the octicons directly: https://octicons.github.com/

**Comment by johndbritton** [comment]
I didn't look at the implementation, definitely prefer using octicons.

**Comment by tarebyte** [comment]
With the bug icon in the header:

screen shot 2015-09-09 at 00 02 30

**Comment by johndbritton** [comment]
:cool:

**Pull request merged**